amplipi/updater/asgi.py

```diff
@@ -282,7 +282,7 @@ def progress_sse(tasks):
           _sse_info(output)
         else:
           logger.warning(f'error: {output}')
-          _sse_error(output)
+          _sse_failed(output)
```

Comment:
this is the only change I am not sure about. Should a failed task fail the update completely?

Reply:
I think this just makes it show failed. I'm unsure if it does more than that?

Reply:
That's correct, but the question is do we encounter errors during updates that should not qualify the update as failed? (Will this make the update seem sketchier than it actually is and prompt people to retry often?) I haven't seen any cases where that's true, but our customer boxes are potentially still in wildly disparate states.

Reply:
One place an error might make sense to be tolerated is in stream installation/update.

Reply:
I'm reverting this change; it only serves to surface issues with the configure.py script, but possibly in cases where it isn't "broken", and gives it the appearance of being broken anyways.